packages/core/src/domain/configuration/transportConfiguration.spec.ts

```diff
@@ -150,63 +138,55 @@ describe('transportConfiguration', () => {
-      expect(configuration.isIntakeUrl(`https://session-replay.browser-intake-datadoghq.com/api/v2/replay?xxx`)).toBe(
-        true
-      )
+    it('should detect intake request US3 site', () => {
+      const configuration = computeTransportConfiguration({ clientToken, site: 'us3.datadoghq.com' }, buildEnv)
+      expect(configuration.isIntakeUrl('https://rum.browser-intake-us3-datadoghq.com/api/v2/rum?xxx')).toBe(true)
+      expect(configuration.isIntakeUrl('https://logs.browser-intake-us3-datadoghq.com/api/v2/logs?xxx')).toBe(true)
+      expect(
+        configuration.isIntakeUrl(`https://session-replay.browser-intake-us3-datadoghq.com/api/v2/replay?xxx`)
+      ).toBe(true)
     })
 
-    it('should detect alternate intake domains for EU site', () => {
-      const configuration = computeTransportConfiguration({ clientToken, useAlternateIntakeDomains: true }, buildEnv)
-      expect(configuration.isIntakeUrl(`https://rum.browser-intake-datadoghq.com${v1IntakePath}?xxx`)).toBe(true)
-      expect(configuration.isIntakeUrl(`https://logs.browser-intake-datadoghq.com${v1IntakePath}?xxx`)).toBe(true)
-      expect(configuration.isIntakeUrl(`https://session-replay.browser-intake-datadoghq.com/api/v2/replay?xxx`)).toBe(
-        true
-      )
+    it('should detect intake request US5 site', () => {
+      const configuration = computeTransportConfiguration({ clientToken, site: 'us5.datadoghq.com' }, buildEnv)
+      expect(configuration.isIntakeUrl('https://rum.browser-intake-us5-datadoghq.com/api/v2/rum?xxx')).toBe(true)
+      expect(configuration.isIntakeUrl('https://logs.browser-intake-us5-datadoghq.com/api/v2/logs?xxx')).toBe(true)
+      expect(
+        configuration.isIntakeUrl(`https://session-replay.browser-intake-us5-datadoghq.com/api/v2/replay?xxx`)
+      ).toBe(true)
     })
 
-    it('should force alternate intake domains for other sites', () => {
-      let configuration = computeTransportConfiguration(
-        { clientToken, site: 'us3.datadoghq.com', useAlternateIntakeDomains: false },
-        buildEnv
-      )
-      expect(configuration.isIntakeUrl(`https://rum.browser-intake-us3-datadoghq.com${v1IntakePath}?xxx`)).toBe(true)
-      expect(configuration.isIntakeUrl(`https://rum-http-intake.logs.us3.datadoghq.com${v1IntakePath}?xxx`)).toBe(false)
-
-      configuration = computeTransportConfiguration(
-        { clientToken, site: 'ddog-gov.com', useAlternateIntakeDomains: false },
-        buildEnv
+    it('should detect intake request gov-cloud site', () => {
+      const configuration = computeTransportConfiguration({ clientToken, site: 'ddog-gov.com' }, buildEnv)
+      expect(configuration.isIntakeUrl('https://rum.browser-intake-ddog-gov.com/api/v2/rum?xxx')).toBe(true)
+      expect(configuration.isIntakeUrl('https://logs.browser-intake-ddog-gov.com/api/v2/logs?xxx')).toBe(true)
+      expect(configuration.isIntakeUrl('https://session-replay.browser-intake-ddog-gov.com/api/v2/replay?xxx')).toBe(
+        true
```

Comment:
what about parametrizing this test now?

```ts
[{site, intakeDomain},...].each(...)
```

Reply:
Can we easily detect which test fails with this strategy?

Reply:
You could do something like:

```ts
[...].forEach(({ site, intakeDomain }) => {
  it(`should detect intake request for ${site} site`, () => {
  })
})
```
so you have the same test cases